remote-processor/CMakeLists.txt

```diff
@@ -26,12 +26,18 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+include (GenerateExportHeader)
+
+add_compiler_export_flags()
+
 add_library(remote-processor SHARED
         Message.cpp
         RequestMessage.cpp
         AnswerMessage.cpp
         RemoteProcessorServer.cpp)
 
+GENERATE_EXPORT_HEADER(remote-processor)
```

Comment:
- coding style
- please group together the stuff related to symbol export (like you did in parameter/CMakeLists.txt)